index.html

```diff
@@ -63,6 +61,161 @@ <h2>I’am a Software Developer</h2>
+        <article class="bcg-container project-card">
+          <h3>Profesional Art Printing Data</h3>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required. has been the industry's standard
+          </p>
+          <ul class="list-container">
+            <li class="list-2"><a href="">html</a></li>
+            <li class="list-2"><a href="">bootstrap</a></li>
+            <li class="list-2"><a href="">Ruby</a></li>
+          </ul>
+        </article>
+        <button class="base-btn">See Project</button>
+        <article class="bcg-container project-card">
+          <h3>Profesional Art Printing Data</h3>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required. has been the industry's standard
+          </p>
+          <ul class="list-container">
+            <li class="list-2"><a href="">html</a></li>
+            <li class="list-2"><a href="">bootstrap</a></li>
+            <li class="list-2"><a href="">Ruby</a></li>
+          </ul>
+        </article>
+        <button class="base-btn">See Project</button>
+        <article class="bcg-container project-card">
+          <h3>Profesional Art Printing Data</h3>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required. has been the industry's standard
+          </p>
+          <ul class="list-container">
+            <li class="list-2"><a href="">html</a></li>
+            <li class="list-2"><a href="">bootstrap</a></li>
+            <li class="list-2"><a href="">Ruby</a></li>
+          </ul>
+        </article>
+        <button class="base-btn">See Project</button>
+        <article class="bcg-container project-card">
+          <h3>Profesional Art Printing Data</h3>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required. has been the industry's standard
+          </p>
+          <ul class="list-container">
+            <li class="list-2"><a href="">html</a></li>
+            <li class="list-2"><a href="">bootstrap</a></li>
+            <li class="list-2"><a href="">Ruby</a></li>
+          </ul>
+        </article>
+        <button class="base-btn">See Project</button>
+        <article class="bcg-container project-card">
+          <h3>Profesional Art Printing Data</h3>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required. has been the industry's standard
+          </p>
+          <ul class="list-container">
+            <li class="list-2"><a href="">html</a></li>
+            <li class="list-2"><a href="">bootstrap</a></li>
+            <li class="list-2"><a href="">Ruby</a></li>
+          </ul>
+        </article>
+        <button class="base-btn">See Project</button>
+      </section>
```

Comment:
- [ ] It is commendable that you have styled this section using `display: flex`. However, the requirements specify that you use `display: grid`. Please change that. (This is just for the main container of the projects section. The individual projects can be styled using flexbox)

Reply:
I have added the grid layout as recommended.